code-samples/creating-your-bot/command-handling/index.js

```diff
@@ -23,16 +23,18 @@ for (const folder of commandFolders) {
 	}
 }
 
-client.once(Events.ClientReady, () => {
-	console.log('Ready!');
+client.once(Events.ClientReady, c => {
+	console.log(`Ready! Logged in as ${c.user.tag}`);
```

Comment:
as said in the other PR i'm really not sure why we introduce this duplicate with this horrible naming convention